specification/cdn/resource-manager/Microsoft.Cdn/stable/2021-06-01/afdx.json

```diff
@@ -4366,13 +4366,27 @@
       "description": "settings for security policy patterns to match",
       "type": "object",
       "properties": {
+        "IsProfileLevel": {
+          "description": "Flag to signal whether the WAF policy association is at the default Profile level.",
+          "enum": [
+            "True",
+            "False"
+          ]
+        },
         "domains": {
           "description": "List of domains.",
           "type": "array",
           "items": {
             "$ref": "#/definitions/ActivatedResourceReference"
           }
         },
+        "routes": {
```

Comment:
It will be a little confusing that we use "routes" keyword. For AFDX resource, routes means pair of (domain, path). If we only use URI path here as the PM doc described, we'd better use array string to present path. Otherwise, it will conflict with securityPolicy.domain property.

Reply:
The intent is to have references to route sub-resources (similar to domains). URI path adds complexity because it may or may not align with any route configurations.